### PR TITLE
Update Vic Admin to remove Imagec logs and clean up error reporting

### LINKED ISSUE #2668: vic-admin imageC.log 404
The introduction of ImageC as a library #2587 moved the imageC logging from it's own file to the docker personality log.  The admin currently displays a 404 for the old (now obsolete) imageC log link.

### LINKED ISSUE #2677: Add identification to error reporting in Vic AdminServer log

### LINKED ISSUE #2708: Inconsistent naming in VIC Admin portal
The title that appears in browser tabs is **VCH Admin**. The name that appears in the links to the logs is **VIC Admin Server**. The service that runs the admin portal is `vicadmin`.

vicadmin

@sgairo and @hickeng, should we settle on **VIC Admin** or **VCH Admin** as the official name for this portal?

Also, neither **VIC Admin** nor **VCH Admin** appears in the main body of the page. Should we change **_VCH_Name_ Admin** to something like **VIC/VCH Admin Portal for  _VCH_Name_**?

### PR DESCRIPTION
**Pull request by sgairo**
Fixes #2708 
Fixes #2677 
Fixes #2668

### CODE REVIEW

**Comment by fdawg4l** [comment]
LGTM

File: lib/vicadmin/validate.go [hunk 1]
```diff
@@ -107,6 +107,8 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
 			v.LicenseIssues = template.HTML(fmt.Sprintf("%s<span class=\"error-message\">%s</span>\n", v.LicenseIssues, err))
 		}
 	}
+	log.Info(fmt.Sprintf("LicenseStatus set to: %s", v.LicenseStatus))
```
Comment (anchal-agrawal) [inline lib/vicadmin/validate.go]: You can replace `log.Info(fmt.Sprintf(...))` with `log.Infof(...)`.

File: lib/vicadmin/validate.go [hunk 2]
```diff
@@ -107,6 +107,8 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
...
+	log.Info(fmt.Sprintf("LicenseIssues set to: %s", v.LicenseIssues))
```
Comment (caglar10ur) [inline lib/vicadmin/validate.go]: Same here

**Comment by caglar10ur** [comment]
LGTM after Info -> Infof change

File: lib/vicadmin/validate.go (hunk 1 above)
Comment (sgairo) [inline lib/vicadmin/validate.go]: will, do thanks!

File: lib/vicadmin/validate.go (hunk 2 above)
Comment (sgairo) [inline lib/vicadmin/validate.go]: done

File: lib/vicadmin/validate.go [hunk 3]
```diff
@@ -134,6 +136,8 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
 		v.NetworkIssues = template.HTML("")
 
 	}
+	log.Info(fmt.Sprintf("NetworkStatus set to: %s", v.NetworkStatus))
+	log.Info(fmt.Sprintf("NetworkIssues set to: %s", v.NetworkIssues))
```
Comment (sgairo) [inline lib/vicadmin/validate.go]: done

**Comment by sgairo** [comment]
ping

File: isos/vicadmin/dashboard.html [hunk 4]
```diff
@@ -72,29 +72,25 @@ <h3 class="card-title">Virtual Container Host Info</h3>
...
-                  <div class="sixty"><a href="/logs/imagec.log">Docker Image Resolution</a></div>
-                  <div class="fourty"><a href="/logs/tail/imagec.log">Live Log</a></div>
-                </div>
-                <div class="row">
                   <div class="sixty"><a href="/logs/port-layer.log">Port Layer Service</a></div>
                   <div class="fourty"><a href="/logs/tail/port-layer.log">Live Log</a></div>
                 </div>
                 <div class="row">
-                  <div class="sixty"><a href="/logs/init.log">VCH Initialization & Startup</a></div>
+                  <div class="sixty"><a href="/logs/init.log">Initialization & Startup</a></div>
```
Comment (hickeng) [inline isos/vicadmin/dashboard.html]: You could make this:
`Initialization & watchdog`

**Comment by hickeng** [comment]
LGTM - minor comment for name tweak of vic-init log

**Comment by anchal-agrawal** [comment]
:shipit:

File: isos/vicadmin/dashboard.html (hunk 4 above)
Comment (sgairo) [inline isos/vicadmin/dashboard.html]: done

**Comment by sgairo** [comment]
:recycle:

**Pull request merged**